scenarios/crowdsecurity/http-probing.yaml

```diff
@@ -3,7 +3,7 @@ type: leaky
 #debug: true
 name: crowdsecurity/http-probing
 description: "Detect site scanning/probing from a single ip"
-filter: "evt.Meta.service == 'http' && evt.Meta.http_status in ['404', '403', '400'] && evt.Parsed.static_ressource == 'false'"
+filter: "evt.Meta.service == 'http' && ((evt.Meta.http_status in ['404', '403', '400'] && evt.Parsed.static_ressource == 'false') || evt.Meta.sub_type == 'permission_denied')"
```

Comment:
I'm not sure we want to take this into account into this scenario ? isn't it redundant with the the http-generic-bf ?